dask/async.py

```diff
@@ -251,7 +253,7 @@ def _execute_task(arg, cache, dsk=None):
         return arg
 
 
-def execute_task(key, task, data, queue, get_id, raise_on_exception=False):
+def execute_task(key, task_info, marshall, unmarshall, get_id, raise_on_exception=False):
```

Comment:
Slight preference to change to `dumps` and `loads`, as these seem more general and common in python. Upon first glance I thought you were using the `marshal` module. If you don't change, it's only one "l" in each.